core/src/main/java/com/cloud/storage/template/SwiftVolumeDownloader.java

```diff
@@ -0,0 +1,397 @@
+//
```

Comment:
Where would this run? On the mgmt server or on the SSVM?

Reply:
This would run on the SSVM